Jenkinsfile

```diff
@@ -233,6 +233,8 @@ stage('Test') {
             sh '''
               adb uninstall io.realm.xamarintests
               adb install io.realm.xamarintests-Signed.apk
+              adb shell pm grant io.realm.xamarintests android.permission.READ_EXTERNAL_STORAGE
```

Comment:
We're targeting a newer android version which comes with runtime permissions. We can just grant them via adb to avoid a UI prompt.